test/functional/search/content-check-negfull-path/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+rm -f web-dir/*/*.tar
+touch target-dir/test-file
```

Comment:
These don't seem to be used for anything and aren't cleaned up in a teardown script so they'll show up in git status as noise after running make check.